cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -716,13 +716,28 @@ pub enum ExtMode {
 }
 
 impl ExtMode {
+    /// Calculate the `ExtMode` from passed bit lengths of the from/to types.
+    pub(crate) fn calculate(from_bits: u16, to_bits: u16) -> Option<ExtMode> {
```

Comment:
`calculate` made me think that there would be some complicated calculation here happening here; how about using a plain `new` here? (or `infer`, or `for` which would read nicely `ExtMode::for(32, 64)`)

Reply:
Yeah, I like `for` or even `from`... ugh, choosing names. I'll go with `for`.

Reply:
`for` is a keyword, so you can't use it as identifier.

Reply:
Then I guess `new` or `from`?

Reply:
Went with `new`.